lms/djangoapps/courseware/tests/test_about.py

```diff
@@ -417,6 +417,32 @@ def test_course_price_is_not_visble_in_sidebar(self):
         self.assertNotIn('<span class="important-dates-item-text">$10</span>', resp.content)
 
 
+@attr(shard=1)
+class AboutSidebarHTMLTestCase(SharedModuleStoreTestCase):
+    """
+    This test case will check the About page for the content in the HTML sidebar.
+    """
+    def setUp(self):
+        super(AboutSidebarHTMLTestCase, self).setUp()
+        self.course = CourseFactory.create()
+
+    def test_html_sidebar_empty(self):
+        url = reverse('about_course', args=[self.course.id.to_deprecated_string()])
```

Comment:
nit: we should avoid using `to_deprecated_string` string in new code. I think we can use course id here
`args=[self.course.id]`